index.html

```diff
@@ -21,7 +21,7 @@
             </div> -->
         </nav>
     </header>
-    <section class="about-me">
+    <section class="welcome-section">
         <div>
             <h1 class="title">
                 Hey there. I’m Mariane
```

Comment:
- [ ] **[OPTIONAL]** As this was supposed to be your very own portfolio, before you build one from scratch, it would be nice to change the greetings to contain your name!